### ISSUE TITLE
How to load old selenium test suite with new selenium IDE for chrome

### ISSUE THREAD

**Issue by manikumary** [SeleniumHQ/selenium-ide#95]
I have old selenium test suite (HTML) which have many other test cases (html). Now, i am trying to migrate them to new selenium ide .  I could see opening old test suite is not there. Everything is changed to project and extension of files are  (.side). Can please guide us the steps to open our old test suite in new selenium IDE for chrome.

Even after creating new project in new IDE i can add empty test suite and empty test case but how can i add old html test case content to new test case in new ide. (Not even copy paste is possible)
It is not straight forward or not even given a workaround of loading old suite or html files. please guide us.

**Comment by corevo** [comment]
It is currently not supported, we need more real world test suite examples to write the correct migration.  
If you have test cases and their corresponding suites that you can upload, we'd be very happy to look over.

**Comment by Heizer1971** [comment]
some examples

https://drive.google.com/open?id=1vfT1J_3wBfAPINgS3cz3qNWROOTKBlFa

**Comment by corevo** [comment]
Thanks, I'll have a look into it.

**Comment by manikumary** [comment]
Hi , did you get a chance to find out how to do the old ide test migrations to new IDE. Is that now clear how old test suite and test cases are available in html

**Comment by corevo** [comment]
We have discussed it, and we have an idea of how to do it.

**Comment by sreekumarthesun** [comment]
I want to run old selenium ide scripts in old firefox browser atleast. I have installed old firefox(Firefox 28.0.dmg) but unable to install selenium ide plugin. 

can you help me

**Comment by corevo** [comment]
The new IDE is for Firefox 56 and later, for older versions use the older IDE.

**Comment by alexisdal** [comment]
i know it's not very constructive but still...
i got an old webapp that we still have to maintain from time to time. for that reason, since it's business critical, we developped a significant selenium IDE test suite (over 4000 test cases) to make sure we would not break anything.

importing the legacy (html) format does not work in the latest selenium IDE (3.4.2).
And after installing the latest 2.x version (2.9.1), the "File > open test suite" does not open the file selection dialog at all... (error message. tested in Firefox 56)

leaving users without any migrating option is... well... not nice.

**Comment by tourdedave** [comment]
@alexisdal Importing legacy html files is supported (see #119 referenced above). If it's not working for you then something is wrong. Please open a new issue and provide details to help us reproduce it.

**Comment by thomaswors** [comment]
@corevo Has there been any traction with exporting the side project files? 

Apica only takes .html so locking things in a .side project has totally hamstrung the usefulness of the new IDE. Love it, but un-usable.

**Comment by corevo** [comment]
@thomaswors if I may interject, html is the legacy format, and side is the format moving forward, thus is it not the IDE that is un-usable, but rather Apica is.  
You should contact them through their support channel, and ask them to support the new IDE's file format.  
To make a clearer comparison, Microsoft changes the doc format to docx in 2007, all the apps that used to take doc now had to develop docx integration, it didn't make Microsoft's Word un-usable, but rather the apps that did not parse docx.  
Bottom line, this is not an IDE issue, but the service that you are using, therefore you should consult them.

**Comment by thomaswors** [comment]
@thomaswors thats a fair point. I thought side was proprietary as a lot of things are enamored with going to json.

A way to export to other formats would be nice regardless

**Comment by thomaswors** [comment]
@corevo  since you have a vested interest in this i think.

I actually got a response with good news

From Apica:

I've checked with our Product team and we do have plans to have the new selenium .side format supported within ASM in 2019. 

For the time being the support team created an internal tool that allows us to convert the .side files to html file that you can then upload to ASM. 

You can send us the .side files and we can send the html file back.
